.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+	"trailingComma": "none",
+	"bracketSpacing": false,
+	"arrowParens": "avoid",
+	"singleQuote": true,
+	"printWidth": 80,
+	"useTabs": true,
```

Comment:
Shouldn't `"useTabs"` be false?

Reply:
`useTabs` is used for indentation using tab instead of spaces during formatting so we need it.

Reply:
Isn't it a standard that we use spaces instead of tabs?( *Vscode* warns me ) And nearly all our codebase is indented with spaces of 2.

Reply:
2 spaces indentation sounds good. Let's set `useTabs` to false then.

Reply:
So, I will change it and run `npm run format`.